math/vecops/inc/LinkDef.h

```diff
@@ -10,17 +10,19 @@
 #ifdef __ROOTCLING__
 
 // TVec related dictionaries
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<double>::Impl_t+;
 #pragma link C++ class ROOT::Experimental::VecOps::TVec<float>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<UInt_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<ULong_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<ULong64_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<UShort_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<UChar_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<Int_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<Long_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<Long64_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<Short_t>::Impl_t+;
-#pragma link C++ class ROOT::Experimental::VecOps::TVec<Char_t>::Impl_t+;
+#pragma link C++ class ROOT::Experimental::VecOps::TVec<double>::Impl_t+;
```

Comment:
Adding `ROOT::Experimental::VecOps::TVec<double>` will enable autoloading of `libVecOps`. This fixes the test failures we see (I tested `./tree/treeplayer/test/dataframe_histomodels`), even without an explicit dependency from `libTreePlayer` to `libVecOps`.

*Both* should happen: users of `TDataFrame` should rely on `libVecOps` being loaded by the linker, *and* autoloading should work.

Reply:
See c668e3c

Reply:
Ok, so I have two questions: how to push to an open PR? I never did that before. And also, can we declare loading of nested types and survive with only what you added here?

Reply:
@amadio regarding your first question, there is a `allow edits from maintainers` checkbox that is probably checked in your case, which allows root maintainers to push to your branch